tests/suites/smoke/build.sh

```diff
@@ -1,6 +1,6 @@
 run_build() {
     OUT=$(make go-build 2>&1 || true)
```

Comment:
Maybe I didn't have enough ☕  yet but wouldn't this command always exit with status 0 if it fails due to the `|| true`?